### ISSUE TITLE
Partitioning - wipefs no device specified 

### ISSUE THREAD

**Issue by juan11perez** [picodotdev/alis#52]
Thank you for this script. I've set up the partion section as below. It completes the boot partion and then it exits with the message wipefs no device specified

```
# partition
DEVICE="/dev/vda" # sata nvme mmc (single)
DEVICE_TRIM="true" # If DEVICE supports TRIM
LVM="false" # True if use LVM for partitioning
PARTITION_ROOT_ENCRYPTION_PASSWORD="" # LUKS encryption key, if LVM will be user LVM on LUKS. Empty for not use LUKS/encyption. Warning: change it!
PARTITION_ROOT_ENCRYPTION_PASSWORD_RETYPE=""
FILE_SYSTEM_TYPE="ext4" # (single)
SWAP_SIZE="2GiB" # (single, not supported in btrfs)
```

**Comment by picodotdev** [comment]
Should be DEVICE="/dev/sda" instead of DEVICE="/dev/vda"?

**Comment by juan11perez** [comment]
@picodotdev thank you very much for your response. The reason is vda is because im testing it on an unraid virtual machine and kvm labels the disks vda as opposed to sda. 
When i tried first time with sda the script did not complete the partition, after I changed it to vda it completed the partitions but it then exits with this error while trying to do LVM which I've selected as false.

**Comment by picodotdev** [comment]
This line checks if the installation is being performed on a SATA device. Try to change sda to vda.

https://github.com/picodotdev/alis/blob/e12afc76b5adcaf0f730ae1d226fb77bd6dd31c8/alis.sh#L246

Later DEVICE_SATA is used to set partition devices.

https://github.com/picodotdev/alis/blob/e12afc76b5adcaf0f730ae1d226fb77bd6dd31c8/alis.sh#L369

**Comment by juan11perez** [comment]
@picodotdev thank you again. That solved it. I selected the plasma desktop, went through the full installation and in the end it exited with this:
![Screenshot 2019-09-24 at 23 19 35](https://user-images.githubusercontent.com/30061229/65555615-fee8d800-df24-11e9-8746-9be5e406016a.png)
nevertheless the installation was pretty much complete. I restarted logged in as my user type systemctl enable sddm; restart and full desktop working.

Fantastico!!! full arch desktop installation with 2 commands.
Thank you very much
